File: C++/uri1045.cc
/* 
 * File:   triangles.cc
 * Author: Gildásio Zeth
 *
 * Created on September 9, 2015, 11:50 AM
 */

#include <string>
#include <iostream>
#include <cstdlib>
#include <cmath>
#include <stdio.h>

using namespace std;

int main()
{
    float A, B, C, a;
    cin >> A >> B >> C;
    float l [] = {A, B, C};
    
    for(int i = 0;i < 2;i++){
        if (l[i] > l[i+1]){
            a = l[i];
            l[i] = l[i+1];
            l[i+1] = a;
        }
    }
    cout << fixed;
    cout.precision(1);
    if (l[2] >= (l[0] + l[1])){cout << "NAO FORMA TRIANGULO" << endl;}
    else{
        if ((l[2]*l[2]) == (l[1]*l[1] + l[0]*l[0])){cout << "TRIANGULO RETANGULO" << endl;}
        else if ((l[2]*l[2]) > (l[1]*l[1] + l[0]*l[0])){cout << "TRIANGULO OBTUSANGULO" << endl;}
        else if ((l[2]*l[2]) < (l[1]*l[1] + l[0]*l[0])){cout << "TRIANGULO ACUTANGULO" << endl;}
        if (A == B && A == C && B == C){cout << "TRIANGULO EQUILATERO" << endl;}
        if ((l[0] == l[1] && (l[2] != l[1] && l[2] != l[0])) || (l[1] == l[2] && (l[0] != l[1] && l[0] != l[2])) || (l[0] == l[2] && (l[1] != l[0] && l[1] != l[2]))){
        cout << "TRIANGULO ISOSCELES" << endl;}
       }
    
return 0;
}

File: URI/uri1045.py
print "URI 1045"
A, B, C = [float(i) for i in raw_input("").split(" ")]

while A <= 0 and B <= 0 and C <= 0:
	A, B, C = [float(i) for i in raw_input("").split(" ")]

l = [A, B, C]
l.sort(reverse=True)
i = 0

if l[0] >= (l[1] + l[2]):
	print "NAO FORMA TRIANGULO"
else:
        if (l[0]*l[0]) == (l[1]*l[1] + l[2]*l[2]):
                print "TRIANGULO RETANGULO"
        elif (l[0]*l[0]) > (l[1]*l[1] + l[2]*l[2]):
                print "TRIANGULO OBTUSANGULO"
                i = 1
        elif (l[0]*l[0]) < (l[1]*l[1] + l[2]*l[2]):
                print "TRIANGULO ACUTANGULO"
                i = 1
        if (l[0] == l[1] == l[2]):
                print "TRIANGULO EQUILATERO"
        elif (l[0] == l[1] and (l[2] != l[1] and l[2] != l[0])) or (l[1] == l[2] and (l[0] != l[1] and l[0] != l[2])) or (l[0] == l[2] and (l[1] != l[0] and l[1] != l[2])):
                print "TRIANGULO ISOSCELES"
